frontend/script.js

```diff
@@ -0,0 +1,103 @@
+'use strict'
+
+let addBtn = document.querySelector('.addBtn')
+let input = document.querySelector('input')
+let props = document.querySelector('.active')
+let addNew = document.querySelector('.add-new')
+let filterSearch = document.querySelector('.filterSearch')
+let displayTodo = document.querySelector('.displayTodo')
+// let btn = document.querySelector('.btn')
+let newArr, li, checkbox, otherBtn, editBtn, deleteBtn;
+let p = document.createElement('p')
+
+
+let init = () => {
+    let inputValue = input.value
+    // newArr = [inputValue];
+
+    li = document.createElement('li')
+    checkbox = document.createElement('input')
+    otherBtn = document.createElement('div')
+    editBtn = document.createElement('button')
+    deleteBtn = document.createElement('button')
+
+
+    otherBtn.id = 'otherBtn'
+    editBtn.className = 'editBtn'
+    deleteBtn.className = 'deleteBtn'
+    checkbox.type = 'checkbox'
+    checkbox.className = 'checkbox'
+
+    editBtn.innerHTML = `<i class="fa-light fa-pencil"></i>`
+    deleteBtn.innerHTML = `<i class="fa-light fa-trash"></i>`
+
+
+    if(inputValue === "" || inputValue.indexOf(' ') >= 0){
```

Comment:
What are you trying to do here with this validation? Because as it stands, the p tag would spit out "This cannot be empty" if you type of sentence with spaces in between. As it stands you can only add words and not sentences